fvtest/porttest/si.cpp

```diff
@@ -835,6 +835,7 @@ TEST(PortSysinfoTest, sysinfo_test_sysinfo_set_limit_CORE_FILE)
 		intptr_t rc = -1;
```

Comment:
shouldn't this be uint32_t to match the APIs?

Reply:
I know it pre-existed your changes.  If it needs to be this type for code before your changes maybe a new variable is required for your new changes.

Reply:
set_limit returns -1 under certain conditions, which is wrong: it should be OMRPORT_LIMIT_UNKNOWN.  Updated.